btrack/napari.yaml

```diff
@@ -19,6 +22,11 @@ contributions:
     - '*.hdf5'
     accepts_directories: false
 
+  writers:
+  - command: btrack.write_hdf
+    layer_types: ["tracks"]
+    filename_extensions: [".h5", ".hdf", ".hdf5"]
```

Comment:
Does this mean that other extension won't be allowed?

Reply:
When it comes to saving that is

Reply:
yeah exactly - napari will offer to use `btrack` for saving only these file types